docs/EN/Setup/New-User-Setup.md

```diff
@@ -0,0 +1,86 @@
+# New User Setup
+To set up your Trio app, touch the settings icon located on the bottom right of the main screen. Follow this step-by-step guide to start using Trio. Each step contains links to information on setting up each part of your Trio app. Use this as your homebase to reference back to as you set up your app.
+<br>
+
+## Step 1: Connect your Devices
+The first step in setting up your Trio app is to connect your devices to the app. 
+
+### Verify Compatibility
+If you have not already, please verify that your devices are compatible:
+
+ 1. [Phones/Watches](../Getting-Started/iphone.md)
+ 2. [Pumps](../Getting-Started/pump.md)
+ 3. [CGMs](../settings/devices/cgm.md)
+<br>
+
+### Connect Devices
+Once you have verified that you are using compatible devices, you can connect them in the Trio app settings menu.
+
+```{image} img/DeviceMenu.png
+:width: 300px
+```
+
+ 1. [Pump](../Configuration/Devices.md#pump)
+ 2. [CGM](../Configuration/Devices.md#cgm)
+ 3. [Watch (optional)](../Configuration/Devices.md#watch)
+<br>
+
+## Step 2: Add Profile Settings
+The next step is to enter your main profile settings: Basal, ISF, CR, and Target Glucose.
+
+:::{important}
+If you use mg/dL, you must first change the 'Glucose Units' before entering your profile settings.
+
+This is found in the Trio Settings menu under 'Preferences'
+:::
+
+ 1. [Basal](../settings/configuration/basalprofile.md)
+ 2. [Insulin Sensitivity Factor (ISF)](../settings/configuration/insulinsensitivities.md)
+ 3. [Carb Ratio (CR)](../settings/configuration/carbratios.md)
+ 4. [Set Target Glucose](../settings/configuration/targetglucose.md)
+<br>
+
+## Step 3: Adjust Safety Limits
+The next step is to adjust the standard safety limits that will allow Trio to function effectively. There are many more settings that are not mentioned in this section. It is not advised to adjust those until you have familiarized yourself with the system by using it for a while.
+
+:::{warning}
+<b>DO NOT ENABLE DYNAMIC SETTINGS YET</b>
+
+It is essential that Trio have enough data to make sound recommendations. This takes a minimum of 7-14 days. It is not advised to enable dynamic settings until BOTH criteria below are met:
```

Comment:
Dynamic only uses the past 10 days of data, with a minimum of 1 day (technically 21 hours). I think the more important part about waiting to turn on dynamic isn't so much waiting for data (since the only data that's collected is TDD) but rather tuning the primary settings and getting comfortable with using the app/oref.